Cargo.toml

```diff
@@ -1,6 +1,26 @@
 [workspace]
-members = [
-    "garde_tests",
-    "garde_derive",
-    "garde",
-]
+members = ["garde", "garde_*"]
```

Comment:
I would prefer to have these explicitly listed out, especially if there aren't that many of them.